cirq-core/cirq/transformers/target_gatesets/compilation_target_gateset.py

```diff
@@ -80,6 +80,16 @@ class CompilationTargetGateset(ops.Gateset, metaclass=abc.ABCMeta):
     which can transform any given circuit to contain gates accepted by this gateset.
     """
 
+    def __init__(
+        self,
+        *gates: Union[Type['cirq.Gate'], 'cirq.Gate', 'cirq.GateFamily'],
+        name: Optional[str] = None,
+        unroll_circuit_op: bool = True,
+        preserve_moment_structure: bool = True,
+    ):
+        super().__init__(*gates, name=name, unroll_circuit_op=unroll_circuit_op)
+        self._preserve_moment_structure = preserve_moment_structure
+
```

Comment:
Add docstring, here and in `CZTargetGateset` constructor

Reply:
done